src/tools/miropt-test-tools/src/lib.rs

```diff
@@ -51,18 +58,21 @@ pub fn files_for_miropt_test(testfile: &std::path::Path, bit_width: u32) -> Vec<
                     .next()
                     .expect("test_name has an invalid extension");
                 let extension = cap.get(1).unwrap().as_str();
+
                 expected_file =
                     format!("{}{}{}", test_name.trim_end_matches(extension), bit_width, extension,);
                 from_file = test_name.to_string();
                 assert!(test_names.next().is_none(), "two mir pass names specified for MIR dump");
                 to_file = None;
+                let temp = test_name.split('.').collect::<Vec<&str>>();
+                passes.push(temp[temp.len() - 3].to_string());
```

Comment:
What does this magic number 3 mean? (Add the answer as a comment to the code)

Reply:
Haha got it, adding

Reply:
Is the comment I added good or is there more info that is missing?

Reply:
Ugh sorry, github didn't send my comment because I used a review

Reply:
O no worries, thanks for the update 🙂